test/e2e_node/disk_eviction_test.go

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
What is the plan for this test? I believe we plan to run this test through Jenkins. If so you would have to modify the Jenkins properties files here: (test/e2e_node/jenkins/*.properties) and pass the thresholds as part of TEST-ARGS.